Sources/TensorFlow/Core/Runtime.swift

```diff
@@ -681,8 +690,12 @@ extension _ExecutionContext {
     // The execution mode is effectively encoded in the TensorOperation.
     // We can use this to switch between different execution modes.
     // TODO: Can we interop between modes?
-    public static func makeOp(_ name: String, _ nOutputs: Int) -> TFTensorOperation {
-        return TFE_Op(name, nOutputs)
+    public static func makeOp(
```

Comment:
Can this be internal/fileprivate?

Reply:
Removed `public` access, but needed to add @usableFromInline due to uses in other places.